.github/workflows/test-lint.yml

```
@@ -12,20 +12,20 @@ jobs:
     timeout-minutes: 6
     steps:
       - name: Set up Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
         with:
           go-version: 1.19
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: technote-space/get-diff-action@v4
         with:
           PATTERNS: |
             **/**.go
             go.mod
             go.sum
-      - uses: golangci/golangci-lint-action@master
+      - uses: golangci/golangci-lint-action@v3
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.49
+          version: v1.52.1
```

Maybe the patch version should be removed as the comment above states that the patch version should be ignored, or otherwise the comment should be removed.

```suggestion
          version: v1.52
```